build/Microsoft.DotNet.Cli.Publish.targets

```diff
@@ -97,11 +97,13 @@
       AccountKey="$(CloudDropAccessToken)"
       AccountName="$(CloudDropAccountName)"
       ContainerName="$(ContainerName)"
-      Items="@(CliVersionBadgeToUpload)" />
+      Items="@(CliVersionBadgeToUpload)"
+      Overwrite="$(OverwriteOnPublish)" />
```

Comment:
It's also an environment variable, and it's currently used in the other call to the `UploadToAzure` task, which is in PublishContent.targets.  It didn't make sense to me to use that variable that call to UploadToAzure and not for this one.